test/src/helpers.cc

```diff
@@ -1,3 +1,4 @@
+#define DEVING_SUBARRAY_PARTITIONER_STORY5342 0
```

Comment:
Please remove all in-dev condition compilations. I don't know what code is relevant for checkin.

Reply:
Done. (I hope.)